docs/modules/user-guide/partials/proc_defining-kubernetes-resources.adoc

```diff
@@ -2,19 +2,28 @@
 = Defining Kubernetes resources
 
 [role="_abstract"]
-Complex deployments can be described using Kubernetes or OpenShift resource lists that can be referenced in the devfile. This makes them a part of the workspace
+Use Kubernetes or OpenShift resource lists to describe complex deployments. Use devfiles to reference Kubernetes or the OpenShift resource lists; referencing them makes them part of the workspace.
 
 .Procedure
 
 [IMPORTANT]
 ====
-* Because a {prod-short} workspace is internally represented as a single deployment, all resources from the Kubernetes or OpenShift list are merged into that single deployment.
+* Because a devfile workspace is internally represented as a single deployment, all resources from the Kubernetes or OpenShift list are merged into that single deployment.
```

Comment:
`{prod-short}` != devfile

Reply:
Didn't I change it back to `{prod-short}` in my recent commit? 

In a later issue, we should change the {prod-short} file to be lowecase, `devfile` not `Devfile`. We don't capitalize devfile unless it starts a sentence. The authoring stacks use `{prod-short}` more than a lot of our other docs, so since we're keeping it here, we'll want to fix this capitalization inconsistency.

Reply:
> Didn't I change it back to {prod-short} in my recent commit?

That's the point. You did, which is wrong. It resolves to "devfiles workspace', which makes no sense in this context. In Che docs, it used to resolve to "Che workspace", which is a thing. The docs were copied without being properly adapted.

> In a later issue, we should change the {prod-short} file to be lower case, devfile not Devfile. We don't capitalize devfile unless it starts a sentence.

Ack.

> The authoring stacks use {prod-short} more than a lot of our other docs

Clearly :/

> so since we're keeping it here, we'll want to fix this capitalization inconsistency.

No, we're not keeping because it make no sense. It's just a left-over from when it was a Che-specific doc.